crates/nu-command/src/filters/update.rs

```diff
@@ -98,6 +98,30 @@ When updating a specific index, the closure will instead be run once. The first
                     Value::test_int(3),
                 ])),
             },
+            Example {
```

Comment:
Actually I don't think we need these examples, because optional cell path is builtin inside nushell engine, not belongs to specific commands